vllm/envs.py

```diff
@@ -49,6 +49,7 @@
     VLLM_XLA_CACHE_PATH: str = os.path.join(VLLM_CACHE_ROOT, "xla_cache")
     VLLM_XLA_CHECK_RECOMPILATION: bool = False
     VLLM_FUSED_MOE_CHUNK_SIZE: int = 64 * 1024
+    VLLM_DISABLE_FUSED_MOE_ACTIVATION_CHUNKING: bool = True
```

Comment:
nit: positive envvars are easier to grok, like "VLLM_ENABLE_FUSED_MOE_ACTIVATION_CHUNKING"